partialWaveFit/pwaLikelihood.h

```diff
@@ -180,9 +182,12 @@ namespace rpwa {
 		bool addAccIntegral(rpwa::ampIntegralMatrix& accMatrix, unsigned int accEventsOverride = 0);
 
 		bool addAmplitude(const rpwa::amplitudeMetadata& meta);
+		bool addAmplitude(const std::vector<const rpwa::amplitudeMetadata*> meta);
 
 		bool finishInit();
 
+		bool setOnTheFlyBinning(const std::map<std::string, std::pair<double, double> >* binningMap, const std::vector<const eventMetadata*> evtMeta);
```

Comment:
Could you write the arguments on separate lines, as with the `getIntegralMatrices` below?